### PR TITLE
added percentage currency

### PR DESCRIPTION
**Pull request by erhanask**

### CODE REVIEW

File: resources/streams/config/currencies.php [hunk 1]
```diff
@@ -1245,8 +1245,15 @@
...
+            'symbol' => '$',
+        ],
+        'PER' => [
+            'name' => 'Percentage',
+            'direction' => 'ltr',
+            'separator' => ',',
+            'point' => '.',
+            'decimals' => 4,
+            'symbol' => '%'
+        ]
```
Comment (muammertopcu) [inline resources/streams/config/currencies.php:1257]: why did you add percentage in currencies? percentage is not a currency.

**Review by muammertopcu — CHANGES_REQUESTED** [review]

File: resources/streams/config/currencies.php (hunk 1 above)
Comment (erhanask) [reply resources/streams/config/currencies.php:1257]: It will be useful for users who wants to show percentage fields instead of price.

Comment (muammertopcu) [reply resources/streams/config/currencies.php:1257]: Why is the percentage shown instead of the price? How can you compare the percentage?
